share/completions/zef.fish

```diff
@@ -0,0 +1,94 @@
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l fetch
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l build
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l /build -d 'Skip the building phase'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l test
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l /test -d 'Skip the testing phase'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l depends
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l /depends -d 'Do not fetch runtime dependencies'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l build-depends
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l /build-depends -d 'Do not fetch build dependencies'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l test-depends
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l /test-depends -d 'Do not fetch test dependencies'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke test fetch build' -l force -d 'Ignore errors'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l force-resolve -d 'Ignore errors'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke fetch' -l force-fetch -d 'Ignore errors'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l force-extract -d 'Ignore errors'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke build' -l force-build -d 'Ignore errors'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke test' -l force-test -d 'Ignore errors'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l force-install -d 'Ignore errors'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke test fetch build' -x -l timeout -d 'Set a timeout (in seconds)'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke fetch' -x -l fetch-timeout -d 'Set a timeout (in seconds)'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -x -l extract-timeout -d 'Set a timeout (in seconds)'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke build' -x -l build-timeout -d 'Set a timeout (in seconds)'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke test' -x -l test-timeout -d 'Set a timeout (in seconds)'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -x -l install-timeout -d 'Set a timeout (in seconds)'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke test fetch' -x -l degree -d 'Number of simultaneous distributions/jobs to process'
```

Comment:
Just "jobs" should be enough, no need to also talk about "distributions" (whatever that is in this context)